joomla/joomla-cms
# [4.0] Patch for incorrect validation message

Pull Request for Issue # https://github.com/joomla/joomla-cms/issues/19724.

### Summary of Changes
Change /media/system/js/validate.js so that an invalidate input get the correct message.


### Testing Instructions
Insert an invalid value in a field. 
For example insert an invalid email address in the email field while installation (see https://github.com/joomla/joomla-cms/issues/19724) or in the email field when you create a new user.


### Expected result
You get a message that says, that your input is invalid.


### Actual result
You get a message that says that the field is empty.


### Documentation Changes Required
No
